client/src/legacy/LeftAndMain.MobileMenuToggle.js

```diff
@@ -21,8 +21,7 @@ $.entwine('ss', function($){
         const state = store.getState();
         const isOpen = state.mobileMenu.isOpen;
         menu
-          .toggleClass('cms-menu--open', isOpen)
-          .attr('aria-expanded', isOpen);
```

Comment:
Shouldn't there be some corresponding code which adds `.attr('aria-expanded', isOpen);` to update `.cms-help__menu`?  Or does that already existt?

Reply:
If you click the help menu button you'll see the `aria-expanded` attribute is already being updated correctly.